tests/storage/utils.py

```diff
@@ -414,6 +414,29 @@ def create_cirros_dv(
         yield dv
 
 
+def create_fedora_dv(
+    namespace,
+    name,
+    storage_class,
+    access_modes=None,
+    volume_mode=None,
+    client=None,
+    dv_size=Images.Fedora.DEFAULT_DV_SIZE,
+):
+    with create_dv(
+        dv_name=f"dv-{name}",
+        namespace=namespace,
+        url=get_http_image_url(image_directory=Images.Fedora.DIR, image_name=Images.Fedora.FEDORA41_IMG),
```

Comment:
Can also use latest here 
```suggestion
        url=f"{get_test_artifact_server_url()}{FEDORA_LATEST['image_path']}",
```

Reply:
Done! Thanks for the suggestion